contributing/community/releases.rst

```diff
@@ -101,6 +101,22 @@ deprecations, you can upgrade to the new major version (e.g. 5.0) without
 effort, because it contains the same features (the only difference are the
 deprecated features, which your project no longer uses).
 
+PHP Compatibility
+-----------------
+
+The **minimum** PHP version is decided for each major Symfony version by consensus
+amongst the :doc:`core team </contributing/code/core_team>` and documented as
```

Comment:
i was also curious if there's some rule maybe? IIUC new SF major supports latest PHP major at time of release, right?

Reply:
I wouldn't dare to predict the min PHP version for Symfony 7, 8 and 9, if that's what you mean. 😉 

Related: symfony/symfony#40389

Reply:
then i predict SF 8 will require PHP9 ;)

Reply:
I'm gonna bookmark this discussion. Let's see how well it ages. 😁